### ISSUE TITLE
Warn users when the meeting is about to end

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#2021]
Originally reported on Google Code with ID 1267

```
When a meeting has a duration, it will end at the given duration.

However, it's easy for the moderator to loose track of time, or for a meeting go longer
than the original duration.

Need to provide notification to the moderator that the meeting is about to end (such
as 5 minutes before) and give them the option of extending the meeting.

This will avoid scenarios where the meeting suddenly ends when the duration is reached.


```

Reported by `ffdixon` on 2012-07-16 22:17:38

**Comment by capilkey** [comment]
Both breakout rooms and the main meeting will show a warning modal when they are 1 min from closing. The part that is missing is the ability to extend the duration which I don't think is available in either room type.

**Comment by ffdixon** [comment]
Implemented in BigBlueButton 2.2 -- a message will appear when the meeting is within 30 minutes of ending.

![image](https://user-images.githubusercontent.com/290351/57977107-cc4f3180-79be-11e9-8e06-3a1dfedc9c46.png)

**Comment by akhil193** [comment]
@ffdixon please share how can I implement it on our server. We are using xenial-22 version. I'm using php api and I tried to set duration but it is not showing when 30 minute is remaining.
